tests/Unit/Activity/ActivityManagerTest.php

```diff
@@ -259,9 +265,12 @@ public function testPublishNewSharedSubmission(int $shareType, string $shareWith
 		$form->setOwnerId('formOwner');
 		$submitterId = 'submittingUser';
 
-		if ($groupUsers === null) {
-			$this->groupManager->expects($this->never())->method('get');
-		} else {
+		if ($shareType === IShare::TYPE_CIRCLE) {
+			$this->circlesService->expects($this->once())
+				->method('getCircleUsers')
+				->with('sharedCircles')
```

Comment:
```suggestion
				->with('sharedCircle')
```

For groups we also only use the singular in the with-method (see line 241)

Reply:
This is also the reason for the failing PHPUnit checks

Reply:
Thank you, typo :see_no_evil: